dashboard/test/factories/lti_integrations.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
```

Comment:
Common practice here is to remove the generated factory file and add any factories to the shared `dashboard/test/factories/factories.rb`